.github/workflows/bindings_ci.yml

```diff
@@ -16,8 +16,73 @@ env:
   CARGO_TERM_COLOR: always
   MATRIX_SDK_CRYPTO_NODEJS_PATH: bindings/matrix-sdk-crypto-nodejs
   MATRIX_SDK_CRYPTO_JS_PATH: bindings/matrix-sdk-crypto-js
+  # keep in sync with uniffi dependency in root Cargo.toml
+  UNIFFI_REV: fdb769b567865d9c5c7c682a18d0c1301a039c85
```

Comment:
Should we update this revision since https://github.com/matrix-org/matrix-rust-sdk/pull/1147/ will be merged?